src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -269,13 +266,22 @@ public void manualSetpoint(double setpoint) {
 	}
 
 	private void initShuffleboard() {
-		launcherBottomFlyWheelTemp =
-				Shuffleboard.getTab("Launcher")
-						.add("bottom Flywheel temp", 0)
-						.withSize(2, 1)
-						.withWidget(BuiltInWidgets.kTextView)
-						.withPosition(0, 3)
-						.getEntry();
+		if (Robot.isDebugMode()) {
+			Shuffleboard.getTab("Launcher")
+					.add(new SparkPIDWidget(launcherAngleOnePIDController, "launcherAnglePID"))
+					.withPosition(2, 0);
+			Shuffleboard.getTab("Launcher")
+					.add(new SparkPIDWidget(launcherTopPIDController, "launcherTopPID"))
+					.withPosition(0, 0);
+			Shuffleboard.getTab("Launcher")
+					.add(new SparkPIDWidget(launcherBottomPIDController, "launcherBottomPID"))
+					.withPosition(1, 0);
+
+			Shuffleboard.getTab("Launcher")
+					.addDouble("Bottom FlyWheel Temp", () -> launcherBottomMotor.getMotorTemperature());
+			Shuffleboard.getTab("Launcher")
+					.addDouble("Top FlyWheel Temp", () -> launcherTopMotor.getMotorTemperature());
```

Comment:
Have we tested the placement of these? Bottom previously had `.withPosition(0, 3)` and `.withSize(2, 1)` and top previously had `.withPosition(2, 3)` and `.withSize(2, 1)`, but without those I'm worried they will be assigned a spot that overlaps with something else.